dfastmi/batch/ConfigurationChecker.py

```diff
@@ -85,6 +85,8 @@ def check_configuration(self,
                 if success and not return_value:
                     return_value = True
         return return_value
+    
```

Comment:
is there a specific reason to add these extra enters?

Reply:
will remove